### DISCUSSION TITLE
[bug]4589以后黑名单失效

### DISCUSSION THREAD

**Discussion by 06ji** [ElderDrivers/EdXposed#729] [General]
手机：小米9
系统：android10 MIUI12
测试拉黑支付宝，还是能加载蚂蚁模块，人脸过不了

4589以后黑名单失效，S版Y版都测试过，4587人脸可过

还有，S版在EDXP管理器显示是未激活，Y版是正常

**Comment by yujincheng08** [comment]
拉黑之后重启过了吗？

**Reply by 06ji** [reply]
试过了，还是能加载模块

**Reply by 06ji** [reply]
> 拉黑之后重启过了吗？

4612 S版 黑名单有效了，不过EDXP管理器还是黄的。。。提示未激活，只有4587的是绿的

这个4612版本，不用黑名单，人脸测试可过，要点三次左右

**Reply by 06ji** [reply]
> 拉黑之后重启过了吗？

2020.12.15 16：45测试：4613 S 黑名单失效，支付宝人脸不过关，EDXP管理还是黄的

**Comment by kotori2** [comment]
只要是模块没加载都可以认为是黑名单有效
另外据说支付宝会检测magisk修改过的文件，跟edxp无关

**Reply by 06ji** [reply]
但是4587可以用人脸
